### PR TITLE
24067 address updates + small tweaks

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*

- bcgov/entity/issues/24067 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-241-wa9g3arv.web.app

File: strr-platform-web/app/pages/platform/dashboard.vue [hunk 1]
```diff
@@ -12,15 +12,14 @@ const addresses = ref<ConnectAccordionItem[]>([])
 const representatives = ref<ConnectAccordionItem[]>([])
 
 onMounted(async () => {
-  // TODO: break this onMounted logic out into dashboard helper utils
   loading.value = true
   await loadPlatform()
   await new Promise((resolve) => { setTimeout(resolve, 1000) })
```
Comment (deetz99) [inline strr-platform-web/app/pages/platform/dashboard.vue:17]: Is this just for dev?

File: strr-platform-web/app/utils/dashboard-helpers.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { validateEmailRfc5322Regex, validateEmailRfc6532Regex } from '~/utils/connect-validation'
 
-export const setApplicaitonHeaderDetails = (t: any) => {
+export const setApplicaitonHeaderDetails = () => {
```
Comment (deetz99) [inline strr-platform-web/app/utils/dashboard-helpers.ts:3]: typo in function name

**Comment by deetz99** [comment]
Looks good overall 👍

File: strr-platform-web/app/pages/platform/dashboard.vue (hunk 1 above)
Comment (kialj876) [reply strr-platform-web/app/pages/platform/dashboard.vue:17]: will remove

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-platform-dev--pr-241-wa9g3arv.web.app

**Pull request merged**